rfhub2/cli/keywords_importer.py

```diff
@@ -276,6 +278,9 @@ def _serialise_args(self, args: List[str]) -> str:
             else ""
         )
 
+    def _serialise_tags(self, tags: tags) -> List[str]:
```

Comment:
@MaciejWiczk  tags is a module. There is a `Tags` class inside which should be used here

Reply:
@pbylicki, it should be alright now. Unfortunately this class does not seem to have string representation, but using `list(tags._tags)` it works. If I missed best solution please let me know.

Reply:
that's ok, I just meant that `tags` parameter is of class `Tags`, not `tags`